plugin-server/src/worker/ingestion/event-pipeline/pluginsProcessEventStep.ts

```diff
@@ -7,13 +7,14 @@ import { EventPipelineRunner } from './runner'
 
 export async function pluginsProcessEventStep(
     runner: EventPipelineRunner,
-    event: PluginEvent
+    event: PluginEvent,
+    processPerson: boolean
```

Comment:
what if we called this `skipStatefulPlugins` instead here and everywhere else (only adding a comment about this to `runner.ts` around personless)?

Reply:
yup deprecated is fine, just more so I was thinking to separate it from "person" concept which there isn't a real connection with